Documentation/api_v3.md

```diff
@@ -0,0 +1,65 @@
+# etcd3 API
+
+TODO: API doc
+
+## KV API Guarantees
+
+etcd is a consistent and durable key value store with mini-transaction(TODO: link to txn doc when we have it) support. The key value store is exposed through the KV APIs. etcd tries to ensure the strongest consistency and durability guarantees for a distributed system. This specification enumerates the KV API guarantees made by etcd.
+
+### APIs to consider
+
+- Read APIs
```

Comment:
Please [use `*` to denote unordered lists in markdown](https://github.com/coreos/docs/blob/master/STYLE.md#preferred-markdown-symbols)

viz:

```
* Item
* Item2
  * subitem
* Item3
```

* Item
* Item2
  * subitem
* Item3